.github/workflows/publish.yml

```diff
@@ -42,20 +42,50 @@ jobs:
         run: |
           python -m twine upload --repository-url https://test.pypi.org/legacy/ dist/*
           python -m twine upload dist/*
+
       - name: Slack notification
         id: slack
         if: success()
         uses: slackapi/slack-github-action@v1.18.0
         with:
           payload: |
             {
-              "text": "Kili SDK released on Pypi: https://pypi.org/project/kili/",
+              "text": "Kili SDK ${{ github.ref_name }} released!\nGitHub: https://github.com/kili-technology/kili-python-sdk/releases\nPyPI: https://pypi.org/project/kili/",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Kili SDK ${{ github.ref_name }} released!\nGitHub: https://github.com/kili-technology/kili-python-sdk/releases\nPyPI: https://pypi.org/project/kili/"
+                  }
+                }
+              ]
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL_PYTHON_SDK }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+
+      - name: create PR to merge release branch and master
+        run: |
+          git checkout release/${{ github.ref_name }}
+          git checkout -b merge_release/${{ github.ref_name }}
+          git merge master
```

Comment:
imho, there will often be conflicts. So it would be fine to not merge master, it would be the responsibility of the merger who will have a warning about the conflicts anyway.